go.mod

```diff
@@ -40,7 +40,7 @@ require (
 	code.cloudfoundry.org/rep v0.0.0-20200325195957-1404b978e31e // indirect
 	code.cloudfoundry.org/rfc5424 v0.0.0-20180905210152-236a6d29298a // indirect
 	code.cloudfoundry.org/tlsconfig v0.0.0-20200131000646-bbe0f8da39b3 // indirect
-	github.com/DataDog/agent-payload v4.55.0+incompatible
+	github.com/DataDog/agent-payload v4.55.1-0.20210129231257-367e91ca1f0f+incompatible
```

Comment:
Are we going to update it once the agent-payload PR is merged and a new release is created?

Reply:
Yes! 👍